HMCL/src/main/resources/assets/lang/I18N.properties

```
@@ -99,31 +99,33 @@ account.methods=Login Type
 account.methods.authlib_injector=authlib-injector
 account.methods.microsoft=Microsoft
 account.methods.microsoft.birth=How to Change Your Account Birth Date
+account.methods.microsoft.code=Code (Automatically Copied)
 account.methods.microsoft.close_page=Microsoft account authorization is now completed.\n\
    \n\
    There are some extra works for us, but you can safely close this tab for now.
 account.methods.microsoft.deauthorize=Deauthorize
-account.methods.microsoft.error.add_family=An adult must add you to a family in order for you to play Minecraft because you are not yet 18 years old.
-account.methods.microsoft.error.add_family_probably=Please check if the age indicated in your account settings is at least 18 years old. If not and you believe this is an error, you can click "How to Change Your Account Birth Date" to learn how to change it.
+account.methods.microsoft.error.add_family=Please click <a href="https://support.microsoft.com/account-billing/837badbc-999e-54d2-2617-d19206b9540a">here</a> to change your account birth date to be over 18 years old, or add your account to a family.
 account.methods.microsoft.error.country_unavailable=Xbox Live is not available in your current country/region.
-account.methods.microsoft.error.missing_xbox_account=Your Microsoft account does not have a linked Xbox account yet. Please click "Create Profile / Edit Profile Name" to create one before continuing.
-account.methods.microsoft.error.no_character=Please ensure you have purchased Minecraft: Java Edition.\nIf you have already purchased the game, a profile may not have been created yet.\nClick "Create Profile / Edit Profile Name" to create it.
-account.methods.microsoft.error.banned=Your account may have been banned by Xbox Live.\nYou can click "Ban Query" for more details.
+account.methods.microsoft.error.missing_xbox_account=Your Microsoft account does not have a linked Xbox account yet. Please click <a href="https://www.minecraft.net/msaprofile/mygames/editprofile">here</a> to link one.
+account.methods.microsoft.error.no_character=Please confirm that you have purchased Minecraft: Java Edition.\n\
+  If you have already purchased it, a game profile may not have been created. Please click <a href="https://www.minecraft.net/msaprofile/mygames/editprofile">here</a> to create a game profile.
+account.methods.microsoft.error.banned=Your account may have been banned by Xbox Live.\n\
+  You can click <a href="https://enforcement.xbox.com/enforcement/showenforcementhistory">here</a> to check the ban status of your account.
 account.methods.microsoft.error.unknown=Failed to log in, error code: %d.
-account.methods.microsoft.error.wrong_verify_method=Please log in using your password on the Microsoft account login page, and do not use a verification code to log in.
+account.methods.microsoft.error.wrong_verify_method=Please log in using your password on the Microsoft account login page. Do not use verification code login.
```

根据 zkitefly 提供的信息 ([使用密码登录微软账户](https://docs.hmcl.net/launcher/use-password-login-microsoft-account.html))，可以改写一下，不局限于验证码。

```suggestion
account.methods.microsoft.error.wrong_verify_method=Failed to log in. Please try logging into your account using PASSWORD instead of other login methods.
```